openhtf/core/phase_executor.py

```diff
@@ -221,6 +222,18 @@ def execute_phase(self, phase):
     # We've been cancelled, so just 'timeout' the phase.
     return PhaseExecutionOutcome(None)
 
+  def __print_timeout_string(self, phase_desc):
```

Comment:
I would call it `_log_timeout_string`. You never really need leading double underscores in python (unless you're implementing a special function like `__init__`) and they make testing harder. Other than that, looks good.

Reply:
OK make changes as you see fit and pull merge it. Thanks!